aptos-move/framework/aptos-stdlib/sources/math_fixed.move

```diff
@@ -137,3 +136,146 @@ module aptos_std::math_fixed {
         assert!((x - y) * mult < x, 0);
     }
 }
+
+module aptos_std::math_fixed64 {
```

Comment:
I think we should make a new file for this. Having two modules in the same file is weird

Reply:
Done